### ISSUE TITLE
Seal cannot download Twitter videos

### ISSUE THREAD

**Issue by mhsamma** [JunkFood02/Seal#905]
### Checklist

- [ ] I'm reporting a bug unrelated to a specific site.
- [X] I've verified that I'm running the [**latest version**](https://github.com/yt-dlp/yt-dlp/releases/latest) of yt-dlp.
- [X] I've verified that I'm running the latest [**stable version**](https://github.com/JunkFood02/Seal/releases/latest/) of Seal or any later [**preview versions**](https://github.com/JunkFood02/Seal/releases).
- [X] I've read the [**Contributing guidelines**](https://github.com/JunkFood02/Seal/blob/main/CONTRIBUTING.md) and [**Code Of Conduct.**](https://github.com/JunkFood02/Seal/blob/main/CODE_OF_CONDUCT.md)
- [X] I've checkd that the site i'm trying to download from is in the [**Supported Sites**](https://github.com/yt-dlp/yt-dlp/blob/master/supportedsites.md) list from yt-dlp
- [X] I understand that the issue will be (ignored/closed) if I intentionally remove or skip any mandatory field.

### Error reports

App version: 1.9.2-(F-Droid) (10924)
Device information: Android 13 (API 33)
Supported ABIs: [arm64-v8a, armeabi-v7a, armeabi]
Yt-dlp version: 2023.06.22

URL: https://twitter.com/TweetsOfCats/status/1674752912445014018?t=ValGlQ2dj61x4TdlkOD87Q&s=19
ERROR: [twitter] 1674752912445014018: 1674752912445014018: Failed to parse JSON (caused by JSONDecodeError("Expecting value in '': line 1 column 1 (char 0)")); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U

### Screenshots & Screen Records

![Screenshot_20230630_213414_Seal](https://github.com/JunkFood02/Seal/assets/77984797/17b9adb9-6f37-47b5-a1c4-72cc720fbbdd)

### Device info

App version: 1.9.2-(F-Droid) (10924)
Device information: Android 13 (API 33)
Supported ABIs: [arm64-v8a, armeabi-v7a, armeabi]
Yt-dlp version: 2023.06.22

**Comment by JunkFood02** [comment]
TL;DR: Use cookies(in the network settings)
https://github.com/yt-dlp/yt-dlp/issues/7473

**Comment by ayesc9000** [comment]
Note that a rate limit for the ability to read tweets was imposed today, so if you use up your limit (which is pretty easy on non-verified accounts), then you will have to use cookies from a separate account that has not been rate limited.

The exact numbers are:
300 for new, non-verified accounts
600 for non-verified accounts
6,000 for verified accounts
